src/rules/no-extraneous-dependencies.js

```diff
@@ -17,6 +17,7 @@ function extractDepFields(pkg) {
     devDependencies: pkg.devDependencies || {},
     optionalDependencies: pkg.optionalDependencies || {},
     peerDependencies: pkg.peerDependencies || {},
+    bundledDependencies: pkg.bundleDependencies || pkg.bundledDependencies || [],
```

Comment:
this still isn't ensuring it's an array?

Reply:
I transform the object to an array now.